### PR TITLE
Proj M20 Branch to Develop Branch

### PR DESCRIPTION
**Pull request by voxparcxls**

### CODE REVIEW

**Comment by voxparcxls** [comment]
Change to check/test:

* Adaptation Workers modal
* External DB configuration

**Comment by wcgunter** [comment]
@jamesfwood @voxparcxls Do we have any "custom adaption actions" that we can test with? It appears to be working but I'm not sure what they are exactly being used for.

**Comment by wcgunter** [comment]
Workers modal and process actions are working as expected

File: install/sql/adaptation.sql.template [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO cws_code (code)
+VALUES (
+'
+__CUSTOM_METHODS_JAVA__
+'
+);
+
+ALTER TABLE cws_worker ADD asg_class VARCHAR(10);
```
Comment (jamesfwood) [inline install/sql/adaptation.sql.template:8]: @voxparcxls what is this asg_class?  is this m20 specific?  maybe we should change that?

Comment (kgrimes2) [reply install/sql/adaptation.sql.template:8]: IDS maintains its own `adaptation.sql.template` file in our adaptation repo. It includes this line. I don't know that you need to add it here, too.

**Comment by voxparcxls** [comment]
Move README to cws-adaptation and add link from main readme

**Review by jamesfwood — APPROVED** [review]

**Comment by zonagit** [comment]
Looks good to me. Thanks

**Review by zonagit — APPROVED** [review]
Looks good to me. Thanks

**Pull request merged**